lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -5495,6 +5498,17 @@ TypeSystemClang::GetNumChildren(lldb::opaque_compiler_type_t type,
 }
 
 CompilerType TypeSystemClang::GetBuiltinTypeByName(ConstString name) {
+  StringRef name_ref = name.GetStringRef();
+  llvm::Regex re("^(unsigned )?_BitInt\\((.*)\\)$");
```

Comment:
Should we wrap this in a `if (!name_ref.empty() && (name_ref[0] == '_' || name_ref[0] == '(')` to avoid compiling the regex every time?